internal/services/invitations/invitation_resource.go

```diff
@@ -174,7 +174,9 @@ func invitationResourceCreate(ctx context.Context, d *schema.ResourceData, meta
 		if status == http.StatusNotFound {
 			return tf.ErrorDiagF(err, "Timed out whilst waiting for new guest user to be replicated in Azure AD")
 		}
-		return tf.ErrorDiagF(err, "Failed to patch guest user after creating invitation")
+		if status != http.StatusForbidden {
+			return tf.ErrorDiagF(err, "Failed to patch guest user after creating invitation")
+		}
```

Comment:
```suggestion
		if status == http.StatusForbidden {
			log.Printf("[WARN] can't patch user to check it's creation")
		} else {
			return tf.ErrorDiagF(err, "Failed to patch guest user after creating invitation")
		}
```

I just found this usage of warn within the code, https://github.com/hashicorp/terraform-provider-azuread/blob/main/internal/services/identitygovernance/access_package_resource_package_association_resource.go#L147
but this could be an option to handle high and low privilege users.